docs/source/get-started/build-and-run-examples.rst

```diff
@@ -104,19 +104,22 @@ basic usage scenarios of |short_name| with DPCPP. Go to
 
            # Navigate to examples directory and build examples
            cd /examples/oneapi/dpc
-           make so example=svm_two_class_thunder_dense_batch # This will compile and run Correlation example using Intel(R) oneAPI DPC++/C++ Compiler
-           make so mode=build			   # This will compile all the examples
+           cmake -G "Unix Makefiles⁮" -DEXAMPLES=svm_two_class_thunder #This would generate makefiles for all svm examples matching passed name
+           make               # This will compile and run generated svm examples
+           cmake -G "Unix Makefiles⁮" -DTARGET_LINK=static #This wouldgenerate make for static version
+           make               # This will compile and run all the examples
 
       .. group-tab:: Windows
 
          .. code-block:: bash
 
             # Navigate to examples directory and build examples
             cd /examples/oneapi/dpc
-            nmake dll example=svm_two_class_thunder_dense_batch+ # This will compile and run Correlation example using Intel(R) oneAPI DPC++/C++ Compiler
-            nmake dll mode=build			     # This will compile all the examples
+           cmake  -G "NMake Makefiles" -DEXAMPLES=svm_two_class_thunder #This would generate makefiles for all svm examples matching passed name
+           nmake             # This will compile and run generated svm examples
+           cmake  -G "NMake Makefiles" -DTARGET_LINK=static #This wouldgenerate make for static version
```

Comment:
```suggestion
           cmake  -G "NMake Makefiles" -DEXAMPLES=svm_two_class_thunder # This would generate makefiles for all svm examples matching passed name
           nmake             # This will compile and run generated svm examples
           cmake  -G "NMake Makefiles" -DTARGET_LINK=static # This wouldgenerate make for static version
```